src/webgpu/api/operation/memory_sync/texture/texture_sync_test.ts

```diff
@@ -101,37 +111,64 @@ export const kAllWriteOps = [
 ] as const;
 export type WriteOp = typeof kAllWriteOps[number];
 
-export const kAllReadOps = [
-  't2b-copy',
-  't2t-copy',
-  'attachment-load',
```

Comment:
'attachment-load' is removed in favor of TODOs for tests that operate within a single pass. Cause we can't actually do attachment load to read only read without storing, I think.

Reply:
We can for depth/stencil, but sgtm